runtime/outer-PEC.js

```diff
@@ -125,6 +143,22 @@ class OuterPEC extends PEC {
     };
   }
 
+  static async _applyResolveRecipe(arc, recipe) {
```

Comment:
(probably not, but) can you use Planner here instead of this new method? could you pass a list of strategies to the `init` method and then call `plan`?

Reply:
It really isn't worth using a strategizer at all here. Pass the recipe directly into ResolveRecipe and grab, then normalize, the result. At the moment there's a bunch of contortions involving InitPopulation that you'll be able to avoid.